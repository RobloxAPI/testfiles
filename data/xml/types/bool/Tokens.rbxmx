#output: model
<roblox version="4">
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLowerCase</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueUpperCase</string>
			<bool name="Value">TRUE</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTitleCase</string>
			<bool name="Value">True</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueMixedCase</string>
			<bool name="Value">tRuE</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueOne</string>
			<bool name="Value">1</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLowerT</string>
			<bool name="Value">t</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueUpperT</string>
			<bool name="Value">T</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueInvalid</string>
			<bool name="Value">invalid</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmpty</string>
			<bool name="Value"></bool>
		</Properties>
	</Item>

	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLowerCase</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseUpperCase</string>
			<bool name="Visible">FALSE</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTitleCase</string>
			<bool name="Visible">False</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseMixedCase</string>
			<bool name="Visible">fAlSe</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseZero</string>
			<bool name="Visible">0</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLowerF</string>
			<bool name="Visible">f</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseUpperF</string>
			<bool name="Visible">F</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseInvalid</string>
			<bool name="Visible">invalid</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmpty</string>
			<bool name="Visible"></bool>
		</Properties>
	</Item>
</roblox>
