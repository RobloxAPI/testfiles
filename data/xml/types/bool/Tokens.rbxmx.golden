Format: rbxmx
Output: model
Data
	Metadata
	Instances
		0
			ClassName: "BoolValue"
			IsService: false
			Reference: 0
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLowerCase"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		1
			ClassName: "BoolValue"
			IsService: false
			Reference: 1
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueUpperCase"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		2
			ClassName: "BoolValue"
			IsService: false
			Reference: 2
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTitleCase"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		3
			ClassName: "BoolValue"
			IsService: false
			Reference: 3
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueMixedCase"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		4
			ClassName: "BoolValue"
			IsService: false
			Reference: 4
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueOne"
			Children
		5
			ClassName: "BoolValue"
			IsService: false
			Reference: 5
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLowerT"
			Children
		6
			ClassName: "BoolValue"
			IsService: false
			Reference: 6
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueUpperT"
			Children
		7
			ClassName: "BoolValue"
			IsService: false
			Reference: 7
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueInvalid"
			Children
		8
			ClassName: "BoolValue"
			IsService: false
			Reference: 8
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmpty"
			Children
		9
			ClassName: "Frame"
			IsService: false
			Reference: 9
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLowerCase"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		10
			ClassName: "Frame"
			IsService: false
			Reference: 10
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseUpperCase"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		11
			ClassName: "Frame"
			IsService: false
			Reference: 11
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTitleCase"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		12
			ClassName: "Frame"
			IsService: false
			Reference: 12
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseMixedCase"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		13
			ClassName: "Frame"
			IsService: false
			Reference: 13
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseZero"
			Children
		14
			ClassName: "Frame"
			IsService: false
			Reference: 14
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLowerF"
			Children
		15
			ClassName: "Frame"
			IsService: false
			Reference: 15
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseUpperF"
			Children
		16
			ClassName: "Frame"
			IsService: false
			Reference: 16
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseInvalid"
			Children
		17
			ClassName: "Frame"
			IsService: false
			Reference: 17
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmpty"
			Children
